pyproject.toml

```diff
@@ -21,6 +21,8 @@ pyspark = "^3.3.1"
 regex = "^2023.5.5"
 urllib3 = "<=2.0"
 sphinxcontrib-bibtex = "^2.5.0"
+zstandard = "^0.21.0"
```

Comment:
it is necessary for OSCAR-2301 which i've used a lot in the experiments.
If you feel that that should not be part of this repo (since you're not trying to do wide generalised but specific to your needs) let me know.
However, other similar datasets are moving towards zstandard so it might be useful.

btw i don't think it HAS to be zstandard(there are multiple alternatives) but thats the one oscar-2301 tells you.